src/components/FilterList.tsx

```diff
@@ -0,0 +1,30 @@
+import { ChangeEvent } from "react";
+
+interface FilterListProps {
+	searchTerm: string;
+	setSearchTerm: (term: string) => void;
+}
+
+export function FilterList({ searchTerm, setSearchTerm }: FilterListProps) {
```

Comment:
Might be best to rename this to something akin to `FilterListInput`; the current name suggests it itself is the list 😄